app/app/src/main/kotlin/com/hedvig/app/feature/chat/viewmodel/ChatViewModel.kt

```diff
@@ -77,7 +79,7 @@ class ChatViewModel(
                   message.fragments.chatMessageFragment,
                 )
             } else {
-              e { "Chat: subscription was not allowed to write" }
```

Comment:
Should this even be a warning? We are setting the `isSubscriptionAllowedToWrite` to false in a lot of places

Reply:
Not sure if a warning is appropriate either. I was hoping wouldn't show for on-call which is the reason why I changed this, because it's a non-actionable trigger there which basically makes it noise that everyone ignores.
If a warning would also trigger it there, I'd be happy to move this even further down to just `i` for `info` so that we still have the data point, but it silences it for people on-call.